gulpfile.js

```diff
@@ -1,125 +1,82 @@
+const isDev = !isProd && !isTest;
  */
-gulp.task('fractal:build', function () {
-	const builder = fractal.web.builder();
-	builder.on('progress', (completed, total) => logger.update(`Exported ${completed} of ${total} items`, 'info'));
-	builder.on('error', err => logger.error(err.message));
-	return builder.build().then(() => {
-		logger.success('Fractal build completed!');
-	});
-});
-
-gulp.task('css:lint', () =>
-	gulp.src(SASS_SRC)
-	.pipe(sassLint())
-	.pipe(sassLint.format()));
-
-gulp.task('css:process', function () {
-	const postcssPipeline = [require('autoprefixer')];
-	if (isProduction) {
-		postcssPipeline.push(require('cssnano'));
-	}
-	return gulp.src(SASS_SRC)
-	.pipe(gulpif(!isProduction, sourcemaps.init()))
-	.pipe(sassGlob())
-	.pipe(sass.sync({
-		precision: 10,
-		includePaths: ['./node_modules']
-	})).on('error', sass.logError)
-	//.pipe(postcss(postcssPipeline))
-	.pipe(gulpif(!isProduction, sourcemaps.write()))
-	.pipe(rename('site.css'))
-	.pipe(gulp.dest('public/stylesheets'));
-});
-
-gulp.task('css:clean', function () {
-	return del(['public/stylesheets']);
-});
-
-gulp.task('css:watch', function () {
-	gulp.watch(SASS_SRC, watchOpt, gulp.series('css'));
-});
-
-//gulp.task('css', gulp.series(gulp.parallel('css:clean', 'css:lint'), 'css:process'));
-gulp.task('css', gulp.series(gulp.parallel('css:clean'), 'css:process'));
-
-gulp.task('fonts:copy', function () {
-	return gulp.src('assets/fonts/**/*')
-	.pipe(gulp.dest('public/assets/fonts'));
-});
-
-gulp.task('fonts:clean', function (done) {
-	return del(['public/assets/fonts'], done);
-});
-
-gulp.task('fonts', gulp.series('fonts:clean', 'fonts:copy'));
-
-gulp.task('fonts:watch', function () {
-	gulp.watch('assets/fonts/**/*', watchOpt, gulp.parallel('fonts'));
-});
-
-gulp.task('build:clean', function () {
-	return del(['dist']);
-});
-
-//gulp.task('assets', gulp.series('svg', gulp.parallel('css', 'js:lint', 'js', 'fonts', 'images')));
-gulp.task('assets', gulp.series(gulp.parallel('css', 'fonts')));
-
-//gulp.task('watch', gulp.parallel('svg:watch', 'css:watch', 'js:watch', 'fonts:watch', 'images:watch'));
-gulp.task('watch', gulp.parallel('css:watch', 'fonts:watch'));
-
-gulp.task('dev', gulp.series('assets', 'fractal:start', 'watch'));
-
-gulp.task('build', gulp.series(function (cb) {
-	isProduction = true;
-	cb();
-}, 'assets', 'build:clean', 'fractal:build'));
 
-gulp.task('default', gulp.series('build'));
+/* 
+We have two goals currently:
+
+1. generate & update (public/stylesheets/site.css) whenever a component scss file is updated 
+    so we can reference it within (components/_preview.hbs) for a real-time preview.
+2. start fractal server and let it do its thing (watch, livereload, build)
+*/
+
+// copy generated.scss with dynamic imports for
+// all component sass files into public/css   so
+// we can use it in /components/_preview.hbs
+function styles() {
+	return src('src/scss/**/*.scss')
+    .pipe(plumber())
```

Comment:
let's go ahead and remove plumber()? We may need it further down the line for better error reporting, but it doesn't look like we're using it right now

Reply:
Agreed and good catch! I'll remove it ....